### ISSUE TITLE
Installation Instructions 

### ISSUE THREAD

**Issue by her** [ryanoasis/vim-devicons#125]
Hey devicons,

I'm pretty confused on how to install vim-devicons and thought I would open this issue with the purpose of understanding the process while clarifying it for the rest of the community–and then rolling this into an updated readme.md for the project. 

### What's Confusing:

[**Unstable-File-Paths**](https://github.com/ryanoasis/nerd-fonts#unstable-file-paths) on **ryanoasis/_nerd-fonts_** suggests using this path:

* `https://github.com/`**`ryanoasis/nerd-fonts/blob/0.6.0/patched-fonts`**`/DroidSansMono/< font_path >.otf` 

[**Font Installation**](https://github.com/ryanoasis/vim-devicons/#font-installation) on **ryanoasis/_vim-devicons_** suggests two paths, one for OS X(shown) and one for Linux: 

* `cd ~/Library/Fonts && curl -fLo DroidSansMonoForPowerlinePlusNerdFileTypes.otf https://raw.githubusercontent.com/`**`ryanoasis/nerd-fonts/master/patched-fonts`**`/DroidSansMono/Droid%20Sans%20Mono%20for%20Powerline%20Plus%20Nerd%20File%20Types.otf` 

Note these discrepancies(bolded) in the two instructed paths. 

**Font Installation** also leads with the instruction: `You basically have to put any font you would like to use into the` **`~/.fonts`** folder.

However, OS X stores it's fonts in **`~/Library/Fonts/`** as well, note this additional discrepancy between the nerd-fonts repository.

### Clarifying 

Clearing up the installation process, as well as making it more central to the top of readme.md would I think make vim-devicons much more approachable as a project and increase it's adoption rate. The project itself is a wonderful concept, and I want all users to be able to easily configure and install it. 

Once I get this going I'll be sure to add these updated instructions to readme.md and submit a PR, I'll also make sure it looks pretty, and fix the broken links I found :+1: 

I'd submit a PR now... but I haven't been able to get it going yet! Thanks devicons 🙂

### tl;dr

How, as an OS X user, on iTerm2, should I update my powerline-patched Source Code Pro font?

**Comment by ryanoasis** [comment]
@her 

Thanks for the greatly detailed info. It is much appreciated. :+1: 

These are great ideas and yeah I see now there are inconsistencies and in some parts possibly not the easiest to follow :disappointed: 

Let me know if you need help setting things up in order to submit a PR (as you can see pretty much all PRs eventually make it in) :+1:

**Comment by her** [comment]
Okay great 🙂 thanks @ryanoasis 

Are you going to be on gitter at all?

**Comment by ryanoasis** [comment]
@her I am sometimes but not very often. If the gitter room becomes more active I will probably be on it more. I once in a while check in on it.

**Comment by her** [comment]
Okay well I added you on Twitter @eislz maybe we can DM there? I just want to be able to clarify ideas before submitting a PR request, but if not that's cool too 😊 I'll submit a PR once I get devicons running.

**Comment by her** [comment]
Hey quick update I got most of it working 🙂

[![https://gyazo.com/e8ed6697624ebaa21bf97253a0a6020f](https://i.gyazo.com/e8ed6697624ebaa21bf97253a0a6020f.png)](https://gyazo.com/e8ed6697624ebaa21bf97253a0a6020f)

But sadly my UTF icon is not an Apple? 

I'm going to spend sometime tonight working on clarifying some of the readme.md

Any ideas how I can correct the Linux logo and make it Apple's?

**Comment by her** [comment]
lol I just noticed you merged a change, it's fixed 😉

[![https://gyazo.com/4cd782f7f5455ba04a1845f7085bc897](https://i.gyazo.com/4cd782f7f5455ba04a1845f7085bc897.png)](https://gyazo.com/4cd782f7f5455ba04a1845f7085bc897)

**Comment by ryanoasis** [comment]
@her I believe a majority of this is now done.

---

One part I see left would be:
> Note these discrepancies(bolded) in the two instructed paths.

Basically I added that 'unstable paths' warning because people were using that in scripts to get the font but the issue is on `master` the file paths have been changing to make them less and less cumbersome.

I think it is probably best to replace `master` in download references with the latest version, that way if it is just copy and pasted but never updated it theoretically will still work.

If you have any other ideas, let me know

**Comment by her** [comment]
Okay for now I'll close this 🙂
